gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFSInputStream.java

```diff
@@ -142,8 +168,9 @@ public synchronized int read(byte[] buf, int offset, int length) throws IOExcept
   @Override
   public synchronized int read(long position, byte[] buf, int offset, int length)
       throws IOException {
+    Stopwatch stopwatch = Stopwatch.createStarted();
     int result = super.read(position, buf, offset, length);
-
+    captureDuration(POSITIONAL_READ_METHOD, stopwatch, result);
```

Comment:
why not include offset, position and length information also in the logs?

Reply:
I don't think it will be useful as we are not interested in tracking each individual request but in accumulation.

Reply:
Position and offset is interesting, especially in the case of Parquet

Reply:
I will stick to capturing the latency and read bytes as those are the one which we are interested in as of now. As far as capturing Request/Responses per API is concerned, we already have that in our `master` and hopefully we will be able to make use of it and deprecate this change (or whole 2.2.x branch) all together.

Reply:
Are you talking about Guls changes? If yes, it has information only at the aggregate level.